supplementary_style_guide/glossary_terms_conventions/product_conventions/eap-conventions/eap-terms-and-usage--a.adoc

```diff
@@ -0,0 +1,23 @@
+= A
+// Should these terms be added to the AMQ conventions.
+[discrete]
+=== ActiveMQ
+
+*Description* Shorthand for "ActiveMQ Artemis," the built-in messaging technology for JBoss EAP.
+
+*Use it?* No
+
+*Incorrect forms*
+
+*See* ActiveMQ Artemis and JBoss EAP Messaging
+
+[discrete]
+=== ActiveMQ Artemis
+
+*Description* The messaging technology used in JBoss EAP.
+
+*Use it?* Restricted.  Only use when describing the technology used to implement messaging in JBoss EAP.
```

Comment:
It looks like two spaces are used after Restricted.

Reply:
This is now fixed based on updates to the jboss-eap.adoc file in https://github.com/redhat-documentation/supplementary-style-guide/pull/97.